### PR TITLE
update Prisma Studio content

### PR DESCRIPTION
**Pull request by ruheni**
"Copy" Data browser content to Prisma Studio page

### CODE REVIEW

File: content/200-concepts/100-components/06-prisma-studio.mdx
Comment (janpio) [inline content/200-concepts/100-components/06-prisma-studio.mdx:1]: Lines 15+16 can go now.

Comment (janpio) [inline content/200-concepts/100-components/06-prisma-studio.mdx:1]: Data Browser page shows more in the right-hand sidebar - let's do this here as well.

**Review by janpio — APPROVED** [review]
see comments

File: content/200-concepts/100-components/06-prisma-studio.mdx
Comment (ruheni) [reply content/200-concepts/100-components/06-prisma-studio.mdx:1]: I've updated the images. Let me know what you think 😄

**Pull request merged**